crates/kilt-dip-primitives/src/merkle/v0.rs

```diff
@@ -0,0 +1,1299 @@
+// KILT Blockchain – https://botlabs.org
```

Comment:
It might be worth splitting this file up.

Reply:
Yes, I wanted to do it in the testing PR, since it makes testing easier. I will probably leave this comment open and reference it in the ongoing testing PR.

Reply:
Will split this up as part of the unit test PR for that component, which is landing soon™!